docs/quickstart-aks.md

```diff
@@ -41,6 +41,12 @@ In this section, you will deploy an Azure Kubernetes Service (AKS) cluster using
   - `AZURE_AKS_CLUSTER` specifies the name of the AKS cluster.
   - `AZURE_AKS_K8S_VERSION` specifies the version of Kubernetes to use for creating the cluster. The [BKPR Kubernetes version support matrix](../README.md#kubernetes-version-support-matrix-for-bkpr-10) lists the base Kubernetes versions supported by BKPR. `az aks get-versions --location ${AZURE_REGION} -o table` lists the versions available in your region.
 
+* Get a list of your subscriptions:
```

Comment:
I don't think we need to have this step here, we already have the command listed under the description of the `AZURE_SUBSCRIPTION_ID` shell variable.